_projects/2018/astropy/ccdproc-optimize.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Optimizing performance of `ccdproc`
+desc: The package [ccdproc](https://ccdproc.readthedocs.io) has a stable API but little effort has been put in to optimizing its speed or memory use. The project is to do both optimizations.
```

Comment:
apparently the css doesn't like the links in the description, so this will need to be rewritten. Also, please cut the description back to a short sentence so it fits to a similar sized box as the other project ideas'